content/07.pathogenesis.md

```diff
@@ -42,8 +41,13 @@ Metro data was also analyzed to estimate the effects of social distancing measur
 R~t~ decreased in Daegu from around 3 to <1 over the period that social distancing measures were introduced.
 In Seoul, R~t~ decreased slightly, but remained close to 1 (and larger than R~t~ in Daegu).
 This highlights that social distancing measures appeared to work to contain the infection in Daegu, but that in Seoul, R~t~ remains above 1, thus secondary outbreaks are possible.
-It also shows the importance of region-specific analysis: the large decline in case load nationwide is mainly due to the Daegu region, and could hide persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do. 
-Similarly in Iran, estimates of R~t~ declined from 4.86 in the first week to 2.1 by the fourth week after the first cases in Iran were reported [@doi:10.1101/2020.03.20.20038422]. The authors attributed this decline to the effects of self-quarantine and government reductions in working time.
+It also shows the importance of region-specific analysis: the large decline in case load nationwide is mainly due to the Daegu region, and could hide persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do.
+In Iran, estimates of R~t~ declined from 4.86 in the first week to 2.1 by the fourth week after the first cases were reported [@doi:10.1101/2020.03.20.20038422].
+In Europe, analysis of 11 countries inferred the dynamics of R~t~ over a time range from the beginning of the outbreak until March 28, 2020, by which point most countries had implemented major interventions (such as school closures, public gathering bans, and stay-at-home orders) [@doi:10.25561/77731].
+Across all countries, the mean R~t~ before interventions began was estimated as 3.87; R~t~ varied considerably, from below 3 in Norway to above 4.5 in Spain.
+After interventions, R~t~ decreased by an average of 64% across all countries, with mean R~t~=1.43.
+The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden (note that this is in part because sweden did not implement social distancing measures on the same scale as other countries).
```

Comment:
```suggestion
The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden (note that this is in part because Sweden did not implement social distancing measures on the same scale as other countries).
```